extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1092,6 +1088,8 @@ export class AmpA4A extends AMP.BaseElement {
     const iframe = /** @type {!HTMLIFrameElement} */(
         createElementWithAttributes(
             /** @type {!Document} */(this.element.ownerDocument), 'iframe', {
+              height: this.creativeSize_.height,
```

Comment:
Is it accurate to say that that will _cause_ the creative to center? The centering occurs due to the CSS styling that is applied regardless.